packages/accounts/src/light-account/accounts/account.ts

```diff
@@ -108,7 +108,10 @@ export async function createLightAccount({
   signer,
   initCode,
   version = defaultLightAccountVersion("LightAccount"),
-  entryPoint = getEntryPoint(chain),
+  entryPoint = getEntryPoint(chain, {
+    version: AccountVersionRegistry["LightAccount"][version]
+      .entryPointVersion as any,
```

Comment:
I couldn't think of a way to do this without the `any` cast. TypeScript can't infer that this type matches the type for the light account version in `version`.

Reply:
is `GetLightAccountVersion` generic on entrypoint version? it could take in the `TEntryPointVersion` which might strongly type them

Reply:
oh wait:
```
TEntryPointVersion extends GetEntryPointForLightAccountVersion<
    "LightAccount",
    TLightAccountVersion
  > = GetEntryPointForLightAccountVersion<"LightAccount", TLightAccountVersion>
```

weird. I'd expect this to allow the inference

Reply:
Yeah, I'm not quite sure why either, but the error message is a bit too complicated to easily decipher 😞.